taskcluster/ci/build-components/kind.yml

```
@@ -68,6 +71,11 @@ task-defaults:
                     - ':{component}:testRelease'
                     - ':{component}:lintRelease'
                     - ':{component}:publish'
+                beta:
+                    - ':{component}:assembleRelease'
```

Nit: I'd add a comment to warn future readers these commands are the correct ones. Something like: 

```suggestion
                beta:
                    # /!\ Beta builds are considered Release in gradle
                    - ':{component}:assembleRelease'
```